### ISSUE TITLE
outputs like was before

### ISSUE THREAD

**Issue by SupinePandora43** [MathewSachin/Captura#545]
![image](https://user-images.githubusercontent.com/36124472/81086076-78ac5f80-8f11-11ea-9516-3aa52adac701.png)
like was before, cuz it allowed only for "Screenshot"
camera and Windows capture can't use that :C

**Comment by kazyamof** [comment]
True. I also prefer the old interface, where the most tools are all in one single screen!
Now I have to open 2 windows to check which microphone I'm using every time, and also video formats...
![image](https://user-images.githubusercontent.com/3179898/81508080-289a1800-92d8-11ea-937d-7a7aca6d8f7a.png)
